Specs/DateTime/English/TimeZoneParser.json

```diff
@@ -870,5 +870,41 @@
         "Length": 6
       }
     ]
+  },
+  {
+    "Input": "I'm at Eastern Daylight Time",
```

Comment:
Don't forget to add specs to EnablePreview and ComplexCalendar. Please also add cases there where a specific time "merges" with one of the new timezone acronyms.